docs/UserGuide.md

```diff
@@ -118,6 +120,34 @@ Example:
 * `task A0123456Z`
     * Lists out the tasks that student (A0123456Z) has.
     
+### Marking an undone task as done for a particular student: `mark`
+
+Marks a specific undone task as done for a particular student.
+
+Format : `mark n/STUDENT_NAME UNDONE_TASK_INDEX` (or) `mark i/STUDENT_ID UNDONE_TASK_INDEX`
```

Comment:
I think we should only use student id since student name might not be unique?

Reply:
Updated to only handle student ID.